msg/PeopleScene.msg

```diff
@@ -0,0 +1,4 @@
+sensor_msgs/Image scene_image               # store scene image which contain people
```

Comment:
I see in the old `FaceList` message there was just the number of faces. Should there also be a field for the number of detected people? I know this eventually should match the length of the list of the `people`, `face_rois`or `body_rois`, but is there any benefit from removing/adding this field?

Reply:
I will add a `num_detected_people` field